casanovo/data/ms_io.py

```diff
@@ -184,8 +184,11 @@ def save(self) -> None:
                 ),
                 1,
             ):
-                filename = os.path.abspath(psm.spectrum_id[0])
+                filename = psm.spectrum_id[0]
                 idx = psm.spectrum_id[1]
+                if Path(filename).suffix == ".mgf" and idx.isnumeric():
+                    idx = f"index={idx}"
```

Comment:
This is a temporary workaround I added to get the integration test to run, since it doesn't look like the `scan_id` field of a `SpectrumDataset` doesn't have the `index=` prefix when a spectrum comes from an mgf file (it has the `scan=` prefix for mzml files though). It seems like this is a DepthCharge issue so a cleaner fix for this would probably have to happen in DepthCharge.